requirements/ingest/vectara.in

```diff
@@ -0,0 +1,2 @@
+-c ../constraints.in
+-c ../base.txt
```

Comment:
add the vectara python package here

Reply:
Nevermind. You can remove this file and the ingest/vectara.txt. 
Looks like there are no vectara specific dependencies.